converged_sound_open_fw_modules_development/FW/intel_common/module_binmaps/aca_module.binmap

```diff
@@ -0,0 +1,35 @@
+module o ACA
+uuid 88881111-2222-3333-4444-555566667777
+name Aca module example
+version_major 0x1
+version_minor 0x0
+version_hotfix 0x0
+version_build 0x0
+affinity_mask MASTER_CORE_AFFINITY
+instance_count +1
+domain_types DP
+type AudClassModule
+stack_size 1000
+
+group text
+section .aca_module.text
+section ep .aca_module.cmi.text
+group rodata
+section .aca_module.rodata
+group bss
+section .aca_module.noload
+
+sched_caps 1 all
+
+mod_cfg 0 0 0 0 4096 500000 180 180 0 5000 0
+
+
+pin in
+stream_type pcm
+sample_rates 16k
+sample_sizes sample_16b
+sample_containers container_16b
+channel_cfg ch_mono
+
+
+
```

Comment:
How are these file parsed and used to build the image ?

Reply:
Do you mean how ```ch_mono``` is parsed? ```Input/Output pins``` section in README. Basically:

"Values for above properties are available in common binmap file:
``<repo_dir>/FW/intel_common/module_binmaps/dsp_fw_common.binmap``"

Reply:
ok, so this is very similar to the toml that rimage uses. We will need to align these configuration formats.